### ISSUE TITLE
[suggestion] make shulkerboxes.sc work with "pick block" function

### ISSUE THREAD

**Issue by billionai** [gnembon/scarpet#154]
Since shulker boxes with "restock" are supposed to be an extension of the player's inventory, I think it would also be really cool if you could use the "pick block" function from minecraft to bring an item from a shulker box to the player's hand. Not sure how feasible it is, but would really help with big projects that uses lots of blocks.

**Comment by Ghoulboy78** [comment]
Unfortunately scarpet can't really detect middle clicking of blocks, cos that's handled completely client side... nice suggestion tho

**Comment by altrisi** [comment]
That's not true. Handling of middle click is actually server-sided. See gnembon's comment in gnembon/fabric-carpet#557.

But yeah, this would probably be an issue for gnembon/fabric-carpet instead, since the event is missing.
